functions/stdlib/at_quick_exit.c

```diff
@@ -0,0 +1,68 @@
+/* at_quick_exit( void (*)( void ) )
+
+   This file is part of the Public Domain C Library (PDCLib).
+   Permission is granted to use, modify, and / or redistribute at will.
+*/
+
+#include <stdlib.h>
+
+#ifndef REGTEST
+
+extern void (*_PDCLIB_quickexitstack[])( void );
+extern size_t _PDCLIB_quickexitptr;
+
+int at_quick_exit( void (*func)( void ) )
+{
+    if ( _PDCLIB_quickexitptr == _PDCLIB_ATEXIT_SLOTS )
+    {
+        return -1;
+    }
+    else
+    {
+        _PDCLIB_quickexitstack[ _PDCLIB_quickexitptr++ ] = func;
+        return 0;
+    }
+}
+
+#endif
+
+#ifdef TEST
+
+#include "_PDCLIB_test.h"
+
+#include <assert.h>
+
+static int flags[ 32 ];
+
+static void counthandler( void )
+{
+    static int count = 0;
+    flags[ count ] = count;
+    ++count;
+}
+
+static void checkhandler( void )
+{
+    int i;
+    for ( i = 0; i < 31; ++i )
```

Comment:
Is there any reason why there are `flags[0 to 31]`, but only `flags[0 to 30]` are being tested?

Reply:
The testcase is almost an exact copy of the one from `at_exit.c`, and I assume it was done this way by mistake. I changed the loop here and it seems to still work fine.

Reply:
I had thought so; please also fix / report issue upstream